WordPress/build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
         maven { url 'https://maven.fabric.io/repo' }
```

Comment:
> Can we change from mavenCentral() to jcenter()? It's the new default for Android Studio

Good catch, looks like it's harmeless, from http://blog.bintray.com/2015/02/09/android-studio-migration-from-maven-central-to-jcenter/ 

"jcenter() is a superset of mavenCentral(), that encompasses many additional repositories and artifacts. "

> they have the PhotoView version 1.2.4 we need: https://bintray.com/pixty/maven/PhotoView/view

Actually, I don't like that much, since it's not the official repo referenced there :( - I'll keep the version I published to our maven repo for now.